forms-flow-web/src/components/Application/table.js

```diff
@@ -101,29 +105,31 @@ export const columns = (
   redirectUrl
 ) => {
   return [
-    {
-      dataField: "id",
-      text: <Translation>{(t) => t("Application Id")}</Translation>,
-      formatter: (cell, row) => linkApplication(cell, row, redirectUrl),
-      headerClasses: "classApplicationId",
-      sort: true,
-      filter: textFilter({
-        delay: 800,
-        placeholder: `\uf002 ${t("Application Id")}`, // custom the input placeholder
-        caseSensitive: false, // default is false, and true will only work when comparator is LIKE
-        className: "icon-search",
-        style: customStyle,
-        getFilter: (filter) => {
-          idFilter = filter;
-        },
-      }),
-    },
+    /*commented below code, for more detail visit below link
```

Comment:
I really like your approach of leaving the pr link for the commented line. I'll follow that in my future prs :)